interop-tests/src/bin/wasm_ping.rs

```diff
@@ -0,0 +1,211 @@
+use std::env;
+use std::process::Stdio;
+use std::time::Duration;
+
+use anyhow::{Context, Result};
+use axum::body;
+use axum::http::{header, Uri};
+use axum::response::{Html, IntoResponse, Response};
+use axum::routing::get;
+use axum::{extract::State, http::StatusCode, routing::post, Json, Router};
+use redis::{AsyncCommands, Client};
+use thirtyfour::prelude::*;
+use tokio::io::{AsyncBufReadExt, BufReader};
+use tower_http::cors::CorsLayer;
+use tower_http::trace::TraceLayer;
+use tracing::warn;
+use tracing_subscriber::{fmt, prelude::*, EnvFilter};
+
+use interop_tests::BlpopRequest;
+
+mod config;
+
+/// Embedded Wasm package
+///
+/// Make sure to build the wasm with `wasm-pack build --target web`
+#[derive(rust_embed::RustEmbed)]
+#[folder = "pkg"]
+struct WasmPackage;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // start logging
+    tracing_subscriber::registry()
+        .with(fmt::layer())
+        .with(EnvFilter::from_default_env())
+        .init();
+
+    // read env variables
+    let config = config::Config::from_env()?;
+    let test_timeout = Duration::from_secs(config.test_timeout);
+    let proxy_addr = env::var("proxy_addr").unwrap_or_else(|_| "127.0.0.1:6378".into());
```

Comment:
Why do we need this? The redis proxy endpoint is on the same host as the JS / WASM that it is served from, right? Shouldn't we be able to make requests with a relative URL and still be able to hit the endpoint?

Reply:
Even if we make requests with absolute URLs, why does it need to be configurable?

Reply:
The webpage itself I'd prefer if we serve on something more common for HTTP like 8080 and not have it configurable.

Reply:
yeah, it's a leftover from when it was yet just a redis proxy

Reply:
Unfortunately `reqwest` doesn't support relative addresses https://github.com/seanmonstar/reqwest/issues/988. As this is just a test I just introduced static address to bind to instead but still pass it to wasm as a param

Reply:
> Unfortunately `reqwest` doesn't support relative addresses [seanmonstar/reqwest#988](https://github.com/seanmonstar/reqwest/issues/988). As this is just a test I just introduced static address to bind to instead but still pass it to wasm as a param

Lol, that moment when you stumble over your own issues on GitHub :joy:

Reply:
hmm, I think this is worth checking https://github.com/hamza1311/reqwasm